bazel/envoy_binary.bzl

```diff
@@ -57,6 +57,7 @@ def _envoy_linkopts():
             # See note here: https://luajit.org/install.html
             "-pagezero_size 10000",
             "-image_base 100000000",
+            "-lresolv",
```

Comment:
this should be added into the `cmake_external` rule as `linkopt`, so any target depends on that will have this linker option.

Reply:
Do any of our cmake targets depend on c-ares?

Reply:
What do you mean? I meant you should add linkopt [here](https://github.com/envoyproxy/envoy/blob/master/bazel/foreign_cc/BUILD#L78) so it propagates.

Reply:
Oh, good point!